app.jsx

```diff
@@ -1,25 +1,23 @@
 import { render } from 'preact'
-
-import { center } from './data/ascii.js'
-import { Header } from './component/header.jsx'
-import { P, divider } from './component/elements.jsx'
 import { Router } from './lib/router.js'
 import { Profile } from './page/profile.jsx'
 import { Home } from './page/home.jsx'
+import {Auth} from './page/auth.jsx';
+import {Evaluations} from './page/evaluations.jsx';
+import {Results} from './page/results.jsx';
+import {Evaluation} from './page/evaluation.jsx';
+import {Quizz} from './page/quizz.jsx';
 
 const App = () => (
-  <>
-    <Header />
-    <Router>
-      <Profile path="/profile" />
-      <Home path="*" />
-    </Router>
-    <footer>
-      {divider}
-      <P fg="comment">{center('Work in progress - NaN - Abidjan')}</P>
-      {divider}
-    </footer>
-  </>
-)
+      <Router>
+          <Auth path="/auth" />
+          <Profile path="/profil" />
+          <Evaluations path="/evaluations" />
+          <Quizz path="/evaluation/quizz"/>
+          <Evaluation path="/evaluation/:evaluationId"/>
```

Comment:
`Quizz` and `Evaluation` components are the same sense about their name. we can't do the difference
you can use a different nomenclature like `Timeline` to specify the result page or something else.